src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -15,8 +15,9 @@
                         <chevron-down-icon size="20" class="icon when-open"></chevron-down-icon>
                     </button>
                 </div>
-                <div class="col-md-3 project-cell">
+                <div class="col-md-3 project-cell d-flex justify-content-between">
                     {{p.name}}
+                    <a @click="renameProject($event, p.id)" href="" v-translate="'renameProject'"></a>
```

Comment:
Could use an edit icon as a link here instead of text "Rename project"

Reply:
Yes, I think that would look neater and you'll probably find that you're starting to run out of columns when you merge this with the latest master. If it's an icon, should include text in tooltip. We're not leaving much room for the project name itself here, which might be quite long depending on how people use it - we should deal gracefully with overflow.